tests/unit/form/TileSelection.spec.js

```diff
@@ -0,0 +1,102 @@
+import { mount } from '@vue/test-utils'
+
+import TileSelection from '@/components/form/TileSelection.vue'
+import TileSelectionOption from '@/components/form/TileSelectionOption.vue'
+
+describe('Form > Tile Selection', () => {
+    it('should have no children by default', async () => {
+        const parent = mount(TileSelection, {
+            props: {
+                modelValue: null
+            },
+            global: {
+                components: {
+                    TileSelectionOption
+                }
+            }
+        })
+
+        expect(parent.findAllComponents('.ff-tile-selection-option').length).toBe(0)
+        expect(parent.vm.children.length).toBe(0)
+    })
+    it('should track registered children', async () => {
+        const mockCheckState = jest.spyOn(TileSelection.methods, 'checkState')
```

Comment:
I'd personally check that the selected state is set correctly (as this is covering implementation details and is more brittle), but I'm happy either way so 👍 regardless.